homeassistant/components/group/config_flow.py

```diff
@@ -67,6 +78,32 @@ async def binary_sensor_options_schema(handler: SchemaCommonFlowHandler) -> vol.
     }
 )
 
+SENSOR_OPTIONS = {
+    vol.Required(CONF_ALL, default=False): selector.BooleanSelector(),
+    vol.Required(CONF_TYPE): selector.SelectSelector(
+        selector.SelectSelectorConfig(options=_STATISTIC_MEASURES),
+    ),
+    vol.Required(CONF_ROUND_DIGITS, default=2): selector.NumberSelector(
+        selector.NumberSelectorConfig(
+            min=0, max=6, mode=selector.NumberSelectorMode.BOX
+        ),
+    ),
```

Comment:
I just remembered a fairly recent meeting in which we discussed adding generic support for configuring precision on sensors for users (on any sensor).

With that in mind, I think we should not add this at this moment maybe?

Reply:
Maybe start now with only adding it to options?
Perhaps not the most critical but if you use average/mean etc. the decimals might be bit concerning for some.

Reply:
> Maybe start now with only adding it to options?

That would later require removing/deprecating it again, resulting in a breaking change.
I would prefer not to have it at all in that case.

Reply:
I got the impression at first it was maybe not so close to implement but got a link to the PR made by @emontnemery so I have now removed rounding from this implementation